src/lib/Scenes/Show/index.tsx

```diff
@@ -6,8 +6,8 @@ import { createFragmentContainer, graphql } from "react-relay"
 
 import { Artists } from "./Components/Artists"
 import { ArtworksContainer as Artworks } from "./Components/Artworks"
-import { Location } from "./Components/Location"
-import ShowHeader from "./Components/ShowHeader"
+import { LocationContainer as Location } from "./Components/Location"
+import { ShowHeaderContainer as ShowHeader } from "./Components/ShowHeader"
```

Comment:
@ashleyjelks @javamonn Let's follow this naming convention for importing Relay Containers which we introduced in reaction